cmd/create/cluster/cmd.go

```diff
@@ -1002,83 +1002,93 @@ func run(cmd *cobra.Command, _ []string) {
 		outputClusterAdminDetails(r, isClusterAdmin, clusterAdminPassword)
 	}
 
+	if isHostedCP && cmd.Flags().Changed("default-mp-labels") {
```

Comment:
Is this change releated to OCM-4801 or do you have a different card for this?

Reply:
It's a minor refactor, this logic was inside the billing block.

As it's unrelated I've extracted for readability.

(No new changes already)

Reply:
https://github.com/openshift/rosa/pull/1599/files#diff-943c0ab57b82c7028652df55e1d49e4f4e4dc41fa094dc4a9b3f2c35887d44b5L1073